pmm/kubernetes-cluster-staging.groovy

```diff
@@ -0,0 +1,372 @@
+                script {
+                    def node = Jenkins.instance.getNode(env.VM_NAME)
+                    Jenkins.instance.removeNode(node)
+                    Jenkins.instance.addNode(node)
+                }
+                archiveArtifacts 'IP'
+            }
+        }
+        stage('Setup Minikube') {
+            steps {
+                script {
+                    withEnv(['JENKINS_NODE_COOKIE=dontKillMe']) {
+                        sh """
+                        export IP=\$(cat IP)
+                        export VM_NAME=\$(cat VM_NAME)
+                        """
+                        node(env.VM_NAME){
+                            sh """
+                                set -o errexit
+                                set -o xtrace
+                                sudo yum -y install curl
+                                sudo curl -Lo /usr/local/sbin/minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+                                sudo chmod +x /usr/local/sbin/minikube
+                                sudo ln -s /usr/local/sbin/minikube /usr/sbin/minikube
+                                alias kubectl='minikube kubectl --'
+                                sleep 5
+                            """
+                        }
+                    }
+                }
+            }
+        }
+        stage('Run Operator Setup Script') {
+            steps {
+                script {
+                    withEnv(['JENKINS_NODE_COOKIE=dontKillMe']) {
+                        sh """
+                            export IP=\$(cat IP)
+                            export VM_NAME=\$(cat VM_NAME)
+                        """
+                        node(env.VM_NAME){
+                            sh """
+                                set -o errexit
+                                set -o xtrace
+                                export PATH=\$PATH:/usr/sbin
+                                minikube version
+                                minikube config set cpus 8
+                                minikube config set memory 29000
+                                minikube config set kubernetes-version 1.16.15
+                                export CHANGE_MINIKUBE_NONE_USER=true
+                                minikube start --driver=none
+                                sudo chown -R $USER $HOME/.kube $HOME/.minikube
+                                sed -i s:/root:$HOME:g $HOME/.kube/config
+                                bash /srv/pmm-qa/pmm-tests/minikube_operators_setup.sh
+                                sleep 10
+                            """
+                        }
+                    }
+                }
+            }
+        }
+        stage('Check Operators Status') {
+            steps {
+                script {
+                    withEnv(['JENKINS_NODE_COOKIE=dontKillMe']) {
+                        sh """
+                            export IP=\$(cat IP)
```

Comment:
IP is not used

Reply:
fixed, https://github.com/Percona-Lab/jenkins-pipelines/pull/744